### PR TITLE
Feature/caes 1656 lists loader

### PR DESCRIPTION
**Pull request by sa715**

### CODE REVIEW

**Review by beshkenadze — APPROVED** [review]

File: packages/common/sagas/workflow.js [hunk 1]
```diff
@@ -976,10 +978,11 @@ function* setWorkInProgressItemSaga({ payload: { item } }) {
 
 function* initDashboardSaga() {
   try {
+    yield put(vaultStartLoading());
     yield takeLatest(VAULTS_ARE_READY, openCurrentVaultSaga);
+    yield put(setWorkInProgressItem(null));
     yield call(initWorkflowSaga);
     yield call(checkTeamsKeyPairs);
-    yield call(checkWIPItem);
```
Comment (cu4xxi) [inline packages/common/sagas/workflow.js:982]: Why did you remove this function call?

Comment (sa715) [reply packages/common/sagas/workflow.js:982]: We need to reset wip-item while changing vault instead of update it

**Review by rtatarinov4xxi — APPROVED** [review]
